### PR TITLE
Add compounds to vapor pressure batch5

### PR DESCRIPTION
**Pull request by sumitnegi7**

### CODE REVIEW

File: src/data/vapor-pressure-constants.ts [hunk 1]
```diff
@@ -1203,6 +1203,291 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    C4: 0.00001305,
+    C5: 2,
+    minimumTemperature: 179.6,
+    maximumTemperature: 445,
+    pressureAtMinimumTemperature: 1.17e2,
+    pressureAtMaximumTemperature: 4.372e6,
+  },
+
+  Difluoromethane: {
+    name: 'Difluoro amine',
```
Comment (qur786) [inline src/data/vapor-pressure-constants.ts:1463]:
```suggestion
    name: 'Difluoromethane',
```

File: src/data/vapor-pressure-constants.ts [hunk 2]
```diff
@@ -1203,6 +1203,291 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    C4: 0.000015065,
+    C5: 2,
+    minimumTemperature: 136.95,
+    maximumTemperature: 351.26,
+    pressureAtMinimumTemperature: 5.43e1,
+    pressureAtMaximumTemperature: 5.761e6,
+  },
+
+  DisopropylAmine: {
+    name: 'Disopropyl amine',
```
Comment (qur786) [inline src/data/vapor-pressure-constants.ts:1478]:
```suggestion
    name: 'Di-sopropyl amine',
```

File: src/data/vapor-pressure-constants.ts [hunk 3]
```diff
@@ -1203,6 +1203,291 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    C3: -7.5868,
+    C4: 0.000015065,
+    C5: 2,
+    minimumTemperature: 136.95,
+    maximumTemperature: 351.26,
+    pressureAtMinimumTemperature: 5.43e1,
+    pressureAtMaximumTemperature: 5.761e6,
+  },
+
+  DisopropylAmine: {
```
Comment (qur786) [inline src/data/vapor-pressure-constants.ts:1477]:
```suggestion
  DisoPropylAmine: {
```
@maneetgoyal @sumitnegi7 should we do this way?

**Review by qur786 — CHANGES_REQUESTED** [review]

File: src/data/vapor-pressure-constants.ts (hunk 3 above)
Comment (sumitnegi7) [reply src/data/vapor-pressure-constants.ts:1477]: The current way is fine i guess it is not separate by space so no need to capitalize it.

File: src/data/vapor-pressure-constants.ts [hunk 4]
```diff
@@ -1203,6 +1203,291 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    C4: 8.2664e-7,
+    C5: 2,
+    minimumTemperature: 282.85,
+    maximumTemperature: 650.15,
+    pressureAtMinimumTemperature: 7.53e2,
+    pressureAtMaximumTemperature: 5.375e6,
+  },
+
+  Dibromomethane: {
+    name: 'Dibromo methane',
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:1238]: name: 'Dibromomethane',

File: src/data/vapor-pressure-constants.ts [hunk 5]
```diff
@@ -1203,6 +1203,291 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    C2: -6661.4,
+    C3: -9.2386,
+    C4: 6.7652e-6,
+    C5: 2,
+    minimumTemperature: 192.5,
+    maximumTemperature: 560,
+    pressureAtMinimumTemperature: 1.72,
+    pressureAtMaximumTemperature: 4.239e6,
+  },
+
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:1371]: Missing `1,2-Dichloropropane`

File: src/data/vapor-pressure-constants.ts [hunk 6]
```diff
@@ -1203,6 +1203,291 @@ export const DIPPRVaporPressureDictionary: Record<
...
+  DiethylSulphide: {
+    name: 'Diethyl sulphide',
+    formula: 'C4H10S',
+    CAS: '352-93-2',
+    C1: 46.705,
+    C2: -5177.4,
+    C3: -3.5985,
+    C4: 107147e-6,
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:1424]: `1.7147E-06`

**Review by saikat-samanta — APPROVED** [review]
LGTM👍

**Review by maneetgoyal — APPROVED** [review]
👍

**Review by qur786 — APPROVED** [review]

**Pull request merged**